src/async/python/async.py

```diff
@@ -0,0 +1,126 @@
+"""Zaber Motion Library advanced async usage example for Python."""
+import asyncio
+from typing import AsyncIterator, List
```

Comment:
Probably doesn't need to import `List` anymore since python3.10 accepts `list` instead.